openwisp_monitoring/device/migrations/0002_create_template.py

```diff
@@ -63,5 +11,10 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunPython(migrate_data, reverse_code=migrations.RunPython.noop)
+        # NOTE: Usage of "Monitoring Template" is deprecated in favour of
+        # "openwisp-openwisp-monitoring" package. See
+        # https://github.com/openwisp/openwrt-openwisp-monitoring#openwrt-openwisp-monitoring
+        migrations.RunPython(
+            migrations.RunPython.noop, reverse_code=migrations.RunPython.noop
+        )
```

Comment:
I have removed the code that was responsible for creating the default monitoring template. 
On new systems, no monitoring template will be created because of this. 

I also thought of commenting the code that creates the template for reference. But, we can always use git to check the older version of the code.

Reply:
🤝